### DISCUSSION TITLE
Creating a semantic token for gradient styles

### DISCUSSION THREAD

**Discussion by dyelax** [chakra-ui/chakra-ui#7346] [Q&A]
I have a few linear gradient styles I'd like to apply throughout the theme. Is there any way to create semantic tokens for these (similar to colors)? If not, is there any way to set a CSS variable at :root that can be accessed by the Chakra theme?

Ideally I'd like to be able to do something like:

```
... {
  variants: {
    primary: { 
      bgGradient: "my-gradient",
    }
  }
}
```

instead of having to write out the full `bgGradient: "linear(to-tl, #9D10CC, #DF1515)"` everywhere it's used

Thanks!

**Comment by Mingyu-Song** [comment]
Maybe you can use layerStyles
